### ISSUE TITLE
Error after starting up

### ISSUE THREAD

**Issue by Slyke** [Luligu/matterbridge-zigbee2mqtt#77]
matterbridge-zigbee2mqtt version: `Update v.2.1.19 to v.2.2.1` (Note: Updating doesn't seem to work, even after restarting MatterBridge, no errors, just it doesn't update).

MatterBridge version: `v.1.5.10`

Here's my error logs:
```
nfo - [19:32:20.206] [PluginManager]: Enabled plugin matterbridge-zigbee2mqtt
info - [19:32:20.209] [PluginManager]: Loading plugin matterbridge-zigbee2mqtt type DynamicPlatform
info - [19:32:20.213] [Matterbridge zigbee2mqtt plugin]: Initializing platform: matterbridge-zigbee2mqtt v
info - [19:32:20.213] [Matterbridge zigbee2mqtt plugin]: Loaded zigbee2mqtt parameters from /root/.matterbridge/matterbridge-zigbee2mqtt.config.json:
info - [19:32:20.214] [Matterbridge zigbee2mqtt plugin]: Connecting to MQTT broker: mqtt://mosquitto.mqtt.svc.cluster.local.:1883
info - [19:32:20.219] [Matterbridge zigbee2mqtt plugin]: MQTT broker at mosquitto.mqtt.svc.cluster.local.:1883 connected
info - [19:32:20.220] [Matterbridge zigbee2mqtt plugin]: MQTT broker at mosquitto.mqtt.svc.cluster.local.:1883 subscribed to: zigbee2mqtt/#
notice - [19:32:20.220] [PluginManager]: Loaded plugin matterbridge-zigbee2mqtt type DynamicPlatform (entrypoint /usr/local/lib/node_modules/matterbridge-zigbee2mqtt/dist/index.js)
info - [19:32:20.220] [PluginManager]: Starting plugin matterbridge-zigbee2mqtt type DynamicPlatform
info - [19:32:20.221] [Matterbridge zigbee2mqtt plugin]: Starting zigbee2mqtt dynamic platform v2.2.1: The plugin has been enabled
info - [19:32:20.260] [Matterbridge zigbee2mqtt plugin]: zigbee2MQTT is online
error - [19:32:30.722] [Matterbridge zigbee2mqtt plugin]: The plugin did not receive zigbee2mqtt bridge info. Check if zigbee2mqtt is running and connected to the MQTT broker.
error - [19:32:30.722] [Matterbridge zigbee2mqtt plugin]: The plugin did not receive zigbee2mqtt bridge devices/groups. Check if zigbee2mqtt is running and connected to the MQTT broker.
error - [19:32:30.723] [PluginManager]: Failed to start plugin matterbridge-zigbee2mqtt: Error: The plugin did not receive zigbee2mqtt bridge state or info or devices/groups. Check if zigbee2mqtt is running and connected to the MQTT broker.
error - [19:32:30.723] [PluginManager]: Plugin matterbridge-zigbee2mqtt not started
```

Here's my config file:
```
{
  "host": "mosquitto.mqtt.svc.cluster.local.",
  "port": 1883,
  "protocolVersion": 5,
  "blackList": [],
  "whiteList": [],
  "switchList": [],
  "lightList": [],
  "outletList": [],
  "featureBlackList": [],
  "postfixHostname": true,
  "debug": false,
  "unregisterOnShutdown": false,
  "name": "matterbridge-zigbee2mqtt",
  "type": "DynamicPlatform",
  "username": "matterbridge",
  "password": "************",
  "topic": "zigbee2mqtt",
  "deviceFeatureBlackList": {}
}
```

I've confirmed that my credentials with by connecting to Mosquitto with MQTT Explorer with these credentials.

**Comment by Luligu** [comment]
Hi,
this "mosquitto.mqtt.svc.cluster.local.",
cannot be a valid host.

Check it.

**Comment by Slyke** [comment]
It's a valid host. I'm running this inside my Kubernetes cluster and that's the service FQDN for Mosquitto.

I'm seeing some logs come through:
```
notice - [19:38:56.542] [Matterbridge]: Package matterbridge-zigbee2mqtt installed correctly
info - [19:38:56.542] [Matterbridge]: Plugin matterbridge-zigbee2mqtt installed. Full restart required.
info - [19:38:56.544] [PluginManager]: Plugin matterbridge-zigbee2mqtt already registered
info - [19:40:44.655] [Matterbridge zigbee2mqtt plugin]: zigbee2MQTT sent 6 devices
info - [19:40:44.657] [Matterbridge zigbee2mqtt plugin]: zigbee2MQTT sent device_announce device: ************* ieee_address: ***********
notice - [20:32:55.052] [Matterbridge]: The plugin matterbridge-zigbee2mqtt is out of date. Current version: 2.1.19. Latest version: 2.2.1.
```

Even though it's saying the plugin is in error state and not started:
![image](https://github.com/user-attachments/assets/830aa5fd-8f9d-4134-a79f-fe2bf5b8b982)

![image](https://github.com/user-attachments/assets/538d1ede-da77-4e8a-8e9c-e24de36b4f34)

**Comment by Luligu** [comment]
It seems that the update is successful...
Try to restart with the debug on file and send me the log please.

**Comment by Teagan42** [comment]
I encountered a problem similar to the error messages in the original issue report. It turned out that the username and password were set to empty strings by default, which messed things up with my EMQX broker since authentication is turned off. Because of that, a user with the username `''` couldn't connect.

I fixed it by changing the defaults to `undefined` and putting together a pull request (#81) to resolve the issue.
